packages/custom-functions-metadata/test/src/testCases.ts

```diff
@@ -70,7 +70,7 @@ describe("test cases", function() {
 
                         // if actual errors are different than expected, write out the actual errors to a file
                         // otherwise, delete the actual errors file if it exists
-                        if (actualErrors !== expectedErrors) {
+                        if (actualErrors && actualErrors !== expectedErrors) {
```

Comment:
@jozefizso What is the reason for this change?

Reply:
The definition of `fs.writeFileSync()` changed in NodeJS 14 and it does not allow `undefined` value.

Without the change the test will fail to compile:

```
TSError: ⨯ Unable to compile TypeScript:
test/src/testCases.ts:74:64 - error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string | Uint8Array | Uint8ClampedArray | Uint16Array | Uint32Array | Int8Array | Int16Array | ... 5 more ... | DataView'.
  Type 'undefined' is not assignable to type 'string | Uint8Array | Uint8ClampedArray | Uint16Array | Uint32Array | Int8Array | Int16Array | ... 5 more ... | DataView'.

74                             fs.writeFileSync(actualErrorsFile, actualErrors);
                                                                  ~~~~~~~~~~~~
```

I think guarding against the `undefined` value is correct change here.

Reply:
Let me look over the code. I think we might want to do the undefined check in a different way...

Reply:
I think it is OK.